content/news/posts/2023/prestashop-9-min-php-version-will-be-81.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "PrestaShop 9 minimum PHP version will be PHP8.1"
+subtitle: "The project moves forward and drops obsolete PHP versions"
+date:   2023-05-12 14:00:00
+authors: [ MathieuFerment ]
+icon: icon-code
+tags:
+ - development
+ - news
+ - 9.0.x
+---
+
+
+The `develop` branch contains the source code that will give birth to PrestaShop 9. This is the future of PrestaShop: it will be released either at the end of the year 2023 or in the beginning of the year 2024 and must carry on the ambition of the project for 2024. This major version must also embrace the change of the PHP ecosystem and consequently it must drop the support of obsolete PHP versions.
```

Comment:
```suggestion
The `develop` branch contains the source code that will give birth to PrestaShop 9. This is the future of PrestaShop: it will be released either at the end of the year 2023 or at the beginning of the year 2024 and must carry on the ambition of the project for 2024. This major version must also embrace the change in the PHP ecosystem, and consequently, it must drop the support of obsolete PHP versions.
```